src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+/*
+ * Association class that records all addition and deletion of tags during a session
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.